.github/workflows/build.yml

```diff
@@ -0,0 +1,56 @@
+on:
+  push:
+    branches:
+      - 'master'
+    tags:
+      - 'v*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+      contents: read
+    env:
+      IMAGE_NAME: bug-buster-machine
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        id: setup_buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Build bounty examples
+        run: make bounties
```

Comment:
One thing that I'm not happy with is the fact that we're building the bounty examples like this, because I'm not sure whether we'll be able to leverage the cache import/export capabilities of GitHub Actions this way...